### ISSUE TITLE
AU-service does not send removed `logo_$_id`-keys anymore

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-autoupdate-service#358]
I sent this request to the AU-service:
```js
{
    "collection": "meeting",
    "ids": [
        2
    ],
    "fields": {
        // [...],
        "id": null,
        "logo_$_id": {
            "type": "template"
        },
        "font_$_id": {
            "type": "template"
        },
        "reference_projector_id": null,
        "is_active_in_organization_id": null,
        // [...]
    }
}
```

I got this response:
```js
{
    // [...]
    "meeting/2/id": 2,
    "meeting/2/is_active_in_organization_id": 1,
    "meeting/2/logo_$_id": [
        "web_header"
    ],
    "meeting/2/logo_$web_header_id": 5,
    // [...]
}
```

So far, so good...

But then, I remove the logo used as `web_header` (this request is regardless, but for the completeness):
```js
[
    {
        "action": "meeting.unset_logo",
        "data": [
            {
                "id": 2,
                "place": "web_header"
            }
        ]
    }
]
```

And I got the following response:
```js
{
    "meeting/2/logo_$_id": []
}
```

You see, the key `web_header` is missing. Therefore, the client does not notice, that the logo changed. 
Is this a bug or is this a result of previously committed changes (like #337)?

**Comment by ostcar** [comment]
This is a result of #337.

When the field `meeting/2/logo_$_id` changes and does not contain the value `web_header` anymore, then the fqfield `meeting/2/logo_$web_header_id` is not requested anymore. So changes to that fqfield (in this case, that it was deleted) are not sent to the client anymore.

Maybe we could think about changing the type of the field meeting/logo_$_id from a template field to a JSON-field. So you could save all logo-values for one meeting into one database key. Then the client could request that keys and gets all updates of this key without the use of the template-feature. The only downside is, that it is not possible to fetch the value of one logo without fetching the values of all other logos. But this should not be a problem.

**Comment by ostcar** [comment]
@GabrielInTheWorld can I close this?

**Comment by GabrielInTheWorld** [comment]
> @GabrielInTheWorld can I close this?

No, not yet. I am still thinking of possible solutions for this behavior.

**Comment by ostcar** [comment]
Since this is not a bug in the autoupdate service (but a problem in some place of the client) I transfer the issue. If the autoupdate service should change something, then please open a new issue in the autoupdate repo.
